### PR TITLE
Update repositories.txt

### PR DESCRIPTION
**Pull request by sswartley**
This is a pull request that would require two manual changes:

The first is to update the URL of a library that is already in the Library Manager. The library (GDXLib) was moved from an individual's GitHub account ("dvernier") to the official GitHub account for Vernier Science Education. 

The second request that would require a manual change is for an update of the libraries name. The original library was misnamed in libraries.properties as "GDXLIb". The name has been corrected and updated as "GDXLib".

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @sswartley. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by sswartley** [comment]
Hello @per1234,

I have not received any notification or update on this request. I wanted to check to see if there was something I needed to do.

Thanks,

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/06/13 15:15:28 Loaded 7062 libraries from DB
Removing GDXLIb
Success!
```

**Comment by sswartley** [comment]
Hi @per1234 

I just wanted to check that there was not an issue with the [GDXLib library](https://github.com/VernierST/GDXLib). After it was moved to a new Github repository and renamed I made some changes to the library. This updated library was released as version 2.0.0. The Library Manager indexer did show it found 2.0.0, but it had errors in the library.properties file and did not update to this version. I then made changes to fix the error and released the updated version as 2.0.1. When I open Arduino IDE version 2.0.1 is available to be installed. However, when I check the log page for the library it does not show that it was updated. 

I am checking the log page at:  https://downloads.arduino.cc/libraries/logs/github.com/VernierST/GDXLib/

Can you confirm that there are no issues with the GDXLib library and that I did not make some mistakes? Thanks!

**Comment by per1234** [comment]
Hi @sswartley. Unfortunately the Arduino web team have instated a draconian caching policy on the indexer log web pages so they are often days out of date.

As you noticed, the actual Library Manager index has been updated even though the logs didn't reflect that. So all is well.

**Comment by sswartley** [comment]
Thank you!

**Pull request merged**
